docs/src/longitudinal.md

```diff
@@ -0,0 +1,121 @@
+# Longitudinal data sets
+
+Longitudinal data are observations made on a set of experimental or observational "units" over time.
+In these data sets we list the response first, then the grouping factor or observational unit, then the time covariate.
+```@setup longitudinal
+using NlmeData: dataset
+using CairoMakie
+using AlgebraOfGraphics
+using DataFrames
+```
+
+## BodyWeight
+```@example longitudinal
+dataset(:BodyWeight)
+```
+Rat weight over time for different diets
+- `weight`: body weight [gm]
+- `rat`: indicator of the rat being weighed
+- `time`: time in the experiment [days]
+- `diet`: one of three diets that the rat receives
+
+The body weights of the rats (in grams) are measured on day 1 and every seven days thereafter until day 64, with an extra measurement on day 44.
+The experiment started several weeks before `day 1.`
+There are three groups of rats, each on a different diet.
+
+### Source
+Hand, D. and Crowder, M. (1996), *Practical Longitudinal Data Analysis*, Chapman and Hall
+
+```@example longitudinal
+let
+    plt = data(dataset(:BodyWeight)) *
+        mapping(
+            :time => "Time in the experiment [days]",
+            :weight => "Body weight [gm]",
+            color = :rat,
+            col = :diet,
+        )
+    draw(plt)
+end
+```
+
+## Cefamandole
+```@example longitudinal
+dataset(:Cefamandole)
+```
+Pharmacokinetics of cefamandole
+- `conc`: observed plasma concentration of cefamandole [μg/ml]
```

Comment:
```suggestion
- `conc`: observed plasma concentration of cefamandole [μg/mL]
```

for consistency with the examples in the repeated-measures file -- [Wikipedia](https://en.wikipedia.org/wiki/Litre#SI_prefixes_applied_to_the_litre) suggests that either `l` or `L` is acceptable